### ISSUE TITLE
Get all open positions 

### ISSUE THREAD

**Issue by bousalem** [JKorf/Binance.Net#564]
Hello everyone,

how to get all opened positions in my accounts with binance.net!!

if someone had docs for binance.net!?

**Comment by AydinIst** [comment]
I need to sample, too.

I tried many ways, but I could not get open position information for futures. 
For ex:
restClient.FuturesUsdt.Order.GetOpenOrders().Data
restClient.FuturesUsdt.Account..GetAccountInfo().Data.Positions etc..

Please provide me correct way to get open position information.

Best regards.

**Comment by bousalem** [comment]
Hi,

Test this code

WebCallResult<IEnumerable<BinanceFuturesPosition>> orders = client.FuturesUsdt.GetPositionInformation();

List<BinanceFuturesPosition> listOrder = orders.Data.Where(e => Decimal.Compare(e.Quantity, 0m) != 0).ToList();

if quantity not null, then you have some position in this symbol

Provide me if that correct code

**Comment by AydinIst** [comment]
Hi.

You are right. Also I found some ways to get all open positions for futures. 

For Example :

`var positionList = restClient.FuturesUsdt.GetPositionInformation().Data.Where(x => x.Quantity != 0).ToList();`

Or

`var positionList = restClient.FuturesUsdt.Account.GetAccountInfo().Data.Positions.Where(x => x.PositionAmount != 0).ToList();`

These codes are working fine for one-way position mode. In one-way mode, return one record. I did not make test for hedge mode.
Probably, in hedge mode, returns two records for buy and long positions.
